Sources/FoundationEssentials/Decimal/Decimal+Conformances.swift

```diff
@@ -445,10 +538,8 @@ extension Decimal : Codable {
 }
 
 // MARK: - SignedNumeric
-// SwiftFoundation's `Decimal` does not fully conform to
-// SignedNumeric yet because no arithmetics have been implemented
 @available(macOS 10.10, iOS 8.0, watchOS 2.0, tvOS 9.0, *)
-extension Decimal /* : SignedNumeric */ {
+extension Decimal : SignedNumeric {
```

Comment:
I presume this addition is for in `swift-foundation` only and not FOUNDATION_FRAMEWORK?